ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -16,7 +16,15 @@
 ### Canonical Types  ###
 ########################
 
-
+SENSOR_SS_ZTM2X:
+  guid: "fbb1219d-e7b4-4bb9-93b4-c41b5c393b65"
+  description: "Dual zone air temperature sensor."
+  is_canonical: true
+  implements:
+  - /SENSOR # Inherits from global namespace
+  - SS
```

Comment:
Looking for some more information on why the sensor needs SS. Is this actually a fan or is this a sensor? If it is a sensor does it actually have a start/stop or does it read values continuously?

Reply:
@ccquigley - thank you, apologies the BMS team have agreed to model the zone temp monitoring separately into 2 units. Closing PR